src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__images {
+  height: 100%;
+  width: 100%;
+}
```

Comment:
You can set width and height attributes for an image directly in HTML. So you dont need to those styles as well as header_logo styles will also be redundand